.devops/azure-pipelines-test-dependent-projects.yml

```diff
@@ -76,12 +81,12 @@ jobs:
   - script: 'git clone https://github.com/Azure/autorest.typescript.git autorest.typescript --recursive --depth 1'
```

Comment:
@daviwil, @joheredi Could it be that we need to use the v4 branch of the autorest.typescript instead of master?
Because master branch is using core-http and we need the version of the code generator that is using ms-rest-js?

Reply:
@joheredi Can you try updating the script to use the v4 branch?

Reply:
Good catch on using the `v4` branch, I should have pulled that over already (it's fixed in the `1.x` branch already).